microcosm_postgres/tests/encryption/fixtures.py

```diff
@@ -0,0 +1,19 @@
+from microcosm.api import binding
+from sqlalchemy import Column, String
+
+from microcosm_postgres.models import EntityMixin, Model
+from microcosm_postgres.store import Store
+from microcosm_postgres.encryption.models import EncryptedMixin
+
+
+class Example(EntityMixin, EncryptedMixin, Model):
```

Comment:
Notice that the only new thing here is the `EncryptedMixin`. We don't really need it since it just adds two columns, but I'm anticipating that the design gets a bit more complicated once encryption becomes conditional:
 - We probably end up with encrypted data living in its own table
 - We probably end up columns in the parent table that include =`plaintext` and an `encrypted_data_id` foreign key, plus a constraint that requires exactly one to be set.
 - We probably end up with some nice mechanism for resolving the `plaintext` value from this choice dynamically.